build/generate_protos.sh

```diff
@@ -5,22 +5,18 @@ set -o pipefail
 shopt -s nullglob
 
 root="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/.."
-xds=${root}/vendor/github.com/envoyproxy/data-plane-api
-
-echo "Expecting protoc version >= 3.5.0:"
-protoc=$(which protoc)
-$protoc --version
```

Comment:
Don't we need this `protoc` version check anymore?

Reply:
Since it's packaged in the docker image, I think it's fine. 3.5.0 is very old by now.